Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 name = "ostree"
 readme = "README.md"
 repository = "https://github.com/ostreedev/ostree-rs"
-version = "0.13.6"
+version = "0.14.0"
```

Comment:
Same concern about semver bump here.  I don't think we did anything requiring that.

Again the way I'd look at this is - if ostree-rs-ext and rpm-ostree compile fine, then that's a strong indication we don't need to bump the semver.

Reply:
The one breaking change I saw is on `RepoCommitTraverseFlags::REPO_COMMIT_TRAVERSE_FLAG_NONE` (though I'm not fully sure why it changed). Our projects are not using it, so they should build regardless.

Reply:
I searched across all of github and gitlab and didn't find any code using this, so it should be fine to go for a `0.13.7` instead as you suggested.

Reply:
> I searched across all of github and gitlab and didn't find any code using this, so it should be fine to go for a 0.13.7 instead as you suggested.

I am now neutral on this topic, so it's up to you!

Reply:
I agree there is a cost in the chain of semver bumps, and this isn't worth that cost. Thanks for the feedback, my brain was on auto-pilot when I initially went through this!